internal/app/repository/models/order.go

```diff
@@ -0,0 +1,15 @@
+package models
+
+import (
+	"database/sql"
+)
+
+type Order struct {
+	ID        int             `json:"id,omitempty"`
+	Number    string          `json:"number"`
+	Status    string          `json:"status"`
+	User      User            `json:"user"`
```

Comment:
Я бы рекомендовал для других сущностей использовать указатель.
```suggestion
	User      *User            `json:"user"`
```